doc/project/checklist.md

```diff
@@ -188,13 +188,13 @@ Prefix "SIM" is applicable for simulation-based DV approach only, while "FPV" is
 
 ### DV_DOC_DRAFT_COMPLETED
 
-- DV document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions, coverage objects (if applicable).
-- Details may be missing since most items are not expected to be fully developed at this stage.
+- DV document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions and the rationale for the chosen functional coverage plan.
+- Details may be missing since most of these items are not expected to be fully understood at this stage.
 
-### DV_PLAN_COMPLETED
+### TESTPLAN_COMPLETED
 
-A fully completed DV plan written in Hjson, indicating the list of planned tests with descriptions indicating the goal of the test and optionally details on stimulus and the checking procedure.
-A fully completed functional coverage plan written in Hjson, indicating the list of functional coverage points and coverage crosses with a description of what feature is covered by this coverpoint.
+A fully completed testplan written in Hjson, indicating the list of planned tests with descriptions indicating the goal of the test and optionally details on stimulus and the checking procedure.
+A detailed functional coverage plan must also be written in the tesplan, indicating the list of coverpoints and crosses with a description of what feature is expected to be covered by each covergroup.
```

Comment:
Nit: tes**t**plan.

Also, I obviously missed this requirement being added (by @rasmus-madsen in d94d69c). To take this seriously, I think we would need to downgrade most (all?) of our blocks to V0! What's the plan here?

Reply:
@rswarbrick Sri and I talked about this. At the time when this requirement went in - the Hjson was not ready for the coverage plan and therefor it did not make sense to downgrade out IP.
(Aes does have a functional coverage plan)

this I agree this is something we should consider - Maybe we don't actually need to downgrade but just ask all to get a coverage plan in and request a review?

Reply:
Yeah, that's probably fine.

Reply:
This PR adds a requirement to have testplan and the dv doc re-reviewed in the V2 stage. We can review the functional coverage requirements during that time. There is no need as such to downgrade all IPs to V0.